sqlx-cli/Cargo.toml

```
@@ -53,7 +52,9 @@ openssl = { version = "0.10.30", optional = true }
 remove_dir_all = "0.7.0"
 
 [features]
-default = ["postgres", "sqlite", "mysql"]
+default = ["postgres", "sqlite", "mysql", "native-tls"]
+rustls = ["sqlx/runtime-async-std-rustls"]
+native-tls = ["sqlx/runtime-async-std-native-tls", "openssl"]
```

```suggestion
native-tls = ["sqlx/runtime-async-std-native-tls"]
```

Enabling the `openssl` crate isn't strictly necessary here and is breaking the build on Windows.